### PR TITLE
Documentation: clarify mind-set of planar rotation

### LINKED ISSUE #3445: Wrong rotation direction in planarRotation functions
`Modelica.Mechanics.MultiBody.Frames.planarRotation` and `Modelica.Mechanics.MultiBody.Frames.TransformationMatrices.planarRotation` either rotate in the opposite direction (respect to right-handed standard) or the documentation is quite unclear.

Given:
- rotation axis along +Z
- rotation angle of 90deg
- a starting vector `v1` along +X

I would expect to get a rotation matrix that rotates the vector `v1` according to `R*v1` in the +Y direction, not in -Y direction as it currently happens.

The expected R for me should have been
`[  0.0, -1.0,  0.0;  
  +1.0,  0.0,  0.0;  
    0.0,  0.0,  1.0]`
instead is  
`[  0.0, +1.0,  0.0;  
  -1.0,    0.0,  0.0;  
    0.0,   0.0,  1.0]`

Am I missing something?

### PR DESCRIPTION
**Pull request by tobolar**
Closes #3445

### CODE REVIEW

File: Modelica/Mechanics/MultiBody/Frames/planarRotation.mo [hunk 1]
```diff
@@ -21,7 +21,7 @@ R = Frames.<strong>planarRotation</strong>(e, angle, der_angle);
 <h4>Description</h4>
 <p>
 The function call <code>Frames.<strong>planarRotation</strong>(e, angle, der_angle)</code> returns
-orientation object R that describes the orientation to rotate in the plane along unit
+orientation object R that describes the orientation of planar rotation along unit
```
Comment (HansOlsson) [inline Modelica/Mechanics/MultiBody/Frames/planarRotation.mo:24]:
```suggestion
orientation object R that describes the orientation of a planar rotation about unit
```
I think English normally has an article ("a" or "the") in these cases; and "along" does for me give the wrong impression - https://en.wikipedia.org/wiki/Rotation_matrix#In_three_dimensions uses "about".
(Writing "orthogonal to" seems excessive.)

Comment (HansOlsson) [reply Modelica/Mechanics/MultiBody/Frames/planarRotation.mo:24]: Or did I completely misunderstand the mindset?

Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Frames/planarRotation.mo:24]: No, you got it perfectly!

**Comment by Harisankar-Allimangalath** [comment]
@MartinOtter is the modification better from your understanding , or do you have any suggestions .

**Review by hubertus65 — APPROVED** [review]

**Pull request merged**